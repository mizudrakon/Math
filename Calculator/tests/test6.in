12
-3.123
12.3 + 3 * 2
x=20.0 / (10-2)
x=20.85 / (5 - 2)
123 + 0.1234567
y=$x
y=$x * $x 
xyz=-(1.25 + $y - ((-3.14) * (2 + 0.98)))
$X * $Y 
 z=3.5
$Z
zz= $xyz * $z
