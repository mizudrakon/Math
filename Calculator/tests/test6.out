-3.123
18.3
2.5
6.95
123.1234567
6.95
48.3025
-58.9097
335.702675
3.5
3.5
-206.18395